test/test_auth.py

```diff
@@ -201,3 +201,117 @@ def test_scram_both(bouncer):
 
     # SCRAM password in userlist.txt
     bouncer.test(dbname="p62", user="scramuser1", password="foo")
+
+
+@pytest.mark.skipif("WINDOWS", reason="Windows does not have SIGHUP")
+def test_connection_with_missing_stats_user_and_enabled_auth_query_without_global_auth_dbname(
```

Comment:
Do we need to specify `enabled_auth_query` in all test names?

Reply:
Tries to highlight the meaning point of these cases, but I can make the function names simpler. I made it because only when `auth_query` enabled pgbouncer falls apart. Anyway we may write the error condition in the function description

Reply:
https://github.com/pgbouncer/pgbouncer/pull/817#discussion_r1149849743
The same as in the previous